website/docs/r/container_node_pool.html.markdown

```diff
@@ -167,10 +167,10 @@ cluster.
 
 The `autoscaling` block supports:
 
-* `min_node_count` - (Required) Minimum number of nodes in the NodePool. Must be >=0 and
+* `min_node_count` - (Required) Minimum number of nodes per instance group in the NodePool. Must be >=0 and
     <= `max_node_count`.
 
-* `max_node_count` - (Required) Maximum number of nodes in the NodePool. Must be >= min_node_count.
+* `max_node_count` - (Required) Maximum number of nodes per instance group in the NodePool. Must be >= min_node_count.
```

Comment:
The min and max node count seem to be per instace group (i.e. per zone for multi-zonal clusters).

https://cloud.google.com/kubernetes-engine/docs/concepts/cluster-autoscaler#minimum_and_maximum_node_pool_size